legal-api/.env.sample

```diff
@@ -89,3 +89,8 @@ BUSINESS_CRED_DEF_ID=
 
 # Digital Wallet Attestation credential
 WALLET_CRED_DEF_ID=
+
+DOCUMENT_API_URL=
+DOCUMENT_API_VERSION=/api/v1
+DOCUMENT_PRODUCT_CODE=LEGAL_API
```

Comment:
Can you set this to the actual entities product code? This is meant to match what is in auth

Reply:
done